python/fledge/common/acl_manager.py

```diff
@@ -191,3 +219,38 @@ async def get_all_entities_for_a_acl(self, acl_name, entity_type):
         else:
             return []
 
+    async def get_acl_for_an_entity(self, entity_name, entity_type):
+        """Get the acl attached to an entity."""
+        q_payload = PayloadBuilder().SELECT("name"). \
+            AND_WHERE(["entity_type", "=", entity_type]). \
+            AND_WHERE(["entity_name", "=", entity_name]).payload()
+        results = await self._storage_client.query_tbl_with_payload('acl_usage', q_payload)
+
+        if len(results['rows']) > 0:
+            for row in results['rows']:
+                return row['name']
+        else:
+            return ""
+
+    async def resolve_pending_notification_for_acl_change(self, svc_name):
+        """Methods that handles the pending notification about acl change to the service."""
+        _logger.debug("svc name {} and pending notifications {}".format(svc_name,
+                                                                       self._pending_notifications))
+        if svc_name not in self._pending_notifications:
+            return
+
+        new_acl = await self.get_acl_for_an_entity(svc_name, "service")
+        old_acl = self._pending_notifications[svc_name]
+
+        _logger.debug("new acl is {} old acl is {}".format(new_acl, old_acl))
+        if new_acl == old_acl and new_acl != "":
+            await self._notify_service_about_acl_change(entity_name=svc_name, acl=new_acl,
```

Comment:
May I suggest to have:

changeACL | updateACL for ACL changed from `foo` to `bar`; and reloadACL when content of  ACL `foo` is changed. This will make things more clear.